### PR TITLE
First cf-layout release

### PR DESCRIPTION
**Pull request by Scotchester**
- http://scotchester.github.io/cf-layout/docs/
- http://scotchester.github.io/cf-layout/demo/

:warning: Needs some short discussion of a few pieces before merging.

### CODE REVIEW

**Comment by Scotchester** [comment]
Notes from review:
- Once cfpb/cf-core#17 is closed, remove inline styles from `img` elements in the docs.
- Remove `content_sidebar-item` in favor of the new general-purpose `block`s with flush modifiers.

**Comment by Scotchester** [comment]
Also, wait for cfpb/cf-core#18.

File: src/cf-layout.less [hunk 1]
```diff
@@ -0,0 +1,1248 @@
...
+/* ==========================================================================
+   Capital Framework
+   cf-grid Enhancements
+
+   The following styles are necessary in this file until such time as they
+   are added to cf-grid.
+   ========================================================================== */
+
+/* topdoc
+  name: cf-grid column dividers
```
Comment (himedlooff) [inline src/cf-layout.less]: I think it's ok to leave this here permanently under a "cf-grid helpers" section or something.

**Comment by himedlooff** [comment]
This is looking pretty good. One observation is that I think we should consider taking `.wrapper` off of `.content_wrapper`, and make `.content_wrapper` extend or call `.wrapper();`. It looks nicer to have this:

```
.content
  .content_wrapper
    .content_main
    .content_sidebar
```

rather than this:

```
.content
  .wrapper.content_wrapper
    .content_main
    .content_sidebar
```

**Comment by Scotchester** [comment]
Okiedoke, I think we're all wrapped up, here! Nice to see our first (I think) good use for `:extend`.

**Comment by himedlooff** [comment]
Looks good thanks!

**Pull request merged**